### PR TITLE
Fix Immersive Interactives

### PR DESCRIPTION
**Pull request by sammorrisdesign-zz**
## What does this change?

Fixes immersive interactives. Imma just merge this in though because us interactives have some stuff to launch soon.

I'm sure there's a better way to do this though. Would love to know if there is.

### CODE REVIEW

**Comment by gtrufitt** [comment]
😵

File: common/app/model/content.scala [hunk 1]
```diff
@@ -833,7 +833,15 @@ final case class Interactive(
     }
   }
 
-  lazy val hasSrcdoc = Jsoup.parseBodyFragment(fields.body).getElementsByTag("iframe").first().hasAttr("srcdoc")
+  lazy val hasSrcdoc = {
+    val iframe = Jsoup.parseBodyFragment(fields.body).getElementsByTag("iframe")
+
+    if (iframe.length > 0) {
```
Comment (rich-nguyen) [inline common/app/model/content.scala]: You can achieve this in scala without length checking:

`val iframes = Jsoup.parseBodyFragment(fields.body).getElementsByTag("iframe")
iframes.headOption.exists(_.hasAttr("srcdoc"))
`

**Comment by rich-nguyen** [comment]
👍

File: common/app/model/content.scala (hunk 1 above)
Comment (sammorrisdesign) [inline common/app/model/content.scala]: Ah sorry I just looked at the option above. That seems way cleaner though.

**Pull request merged**